airbyte-webapp/package.json

```diff
@@ -17,7 +17,7 @@
     "@hot-loader/react-dom": "16.13.0",
     "@papercups-io/chat-widget": "^1.1.5",
     "@papercups-io/storytime": "^1.0.6",
-    "@rest-hooks/legacy": "^1.0.12",
+    "@rest-hooks/legacy": "^2.0.0",
```

Comment:
We have faced an issue before, that old legacy hook was taking into considerations params when returning `loading` prop. So when prop was changed - the request was fired, but loading was always false. Is it fixed in this version? If yes - it could help us a lot :)

Reply:
I believe so; making this work consistently with the behavior of useResource() was a big part of the invalidation/deletion rework

Reply:
https://github.com/coinbase/rest-hooks/blob/master/packages/legacy/src/index.ts#L41 which is identical in logic to https://github.com/coinbase/rest-hooks/blob/master/packages/core/src/react-integration/hooks/useResource.ts#L52

Reply:
Added a test to confirm it works now: https://github.com/coinbase/rest-hooks/pull/488

Feel free to report any issues like this in the future :)

Reply:
I just thought of something regarding this - if it's already in the cache even tho a refetch (for freshness) may fire, it won't mark as loading since the data is already there. Do you maybe expect it to say loading even if the data is already there? (in which case you can use https://resthooks.io/docs/api/Endpoint#invalidifstale-boolean)